src/style.css

```
@@ -1,3 +1,81 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
 body {
+  margin: 0 auto;
+}
+
+header {
+  min-width: 1024px;
+  width: 100%;
+  margin: 0;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
```

```suggestion
header {
  min-width: 1024px;
  width: 100%;
  margin: 0;
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
  padding: 0 50px;
}
```
just add here the padding